src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/FormattingVisitor.cs

```diff
@@ -127,8 +125,10 @@ public override void VisitMarkupElement(MarkupElementSyntax node)
     {
         Visit(node.StartTag);
 
-        // Temporary fix to not break the default Html formatting behavior. Remove after https://github.com/dotnet/aspnetcore/issues/25475.
-        if (!string.Equals(node.StartTag?.Name?.Content, HtmlTagName, StringComparison.OrdinalIgnoreCase))
+        // Void elements, like <meta> or <input> which don't need an end tag don't cause indentation
+        var causesIndentation = node.StartTag is null || !node.StartTag.IsVoidElement();
```

Comment:
❓ Is it valid html to have an end tag on them?

Reply:
Isn't everything valid html? 😛

If you do `<meta></meta>` you get a compiler error from the Razor compiler complaining about it, the Html LSP server issues a diagnostic about it, and if you ignore all of those, the Html formatter doesn't indent the content anyway.

Reply:
😅

Thanks for explaining